modules/anti-ghostping/events/messageCreate.js

```diff
@@ -5,6 +5,8 @@ module.exports.run = async function (client, msg) {
     if (msg.guild.id !== client.config.guildID) return;
     const moduleConfig = client.configurations['anti-ghostping']['config'];
     if (moduleConfig.ignoredChannels.includes(msg.channel.id)) return;
+    if (msg.member.roles.cache.find(r => moduleConfig.ignoredRoles.includes(r.id))) return;
+    if (moduleConfig.ignoredRoles.includes(msg.channel.id)) return;
```

Comment:
Why are we checking if the channel id is part of the ignored roles config setting 😅